rubygem/lib/zeus.rb

```diff
@@ -60,7 +60,7 @@ def boot_steps(identifier)
           master.send_io(remote)
 
           # Now I need to tell the master about my PID and ID
-          local.write "P:#{Process.pid}:#{identifier}\0"
+          local.write "P:#{Process.pid}:#{@parent_pid || 0}:#{identifier}\0"
```

Comment:
The Go binary is packaged with the gem (lurky!) so I'd hope not... though I
imagine you could do it if you tried.

On Tue, Jun 28, 2016 at 4:14 PM, Paul Tarjan <notifications@github.com>
wrote:

> In rubygem/lib/zeus.rb
> <https://github.com/burke/zeus/pull/565#discussion_r68860743>:
>
> > @@ -60,7 +60,7 @@ def boot_steps(identifier)
> >            master.send_io(remote)
> >
> >            # Now I need to tell the master about my PID and ID
> > -          local.write "P:#{Process.pid}:#{identifier}\0"
> > +          local.write "P:#{Process.pid}:#{@parent_pid || 0}:#{identifier}\0"
>
> do people ever upgrade just the ruby gem and not get a new go binary?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/burke/zeus/pull/565/files/b9afd7559e01880bdaba6b497f1bb72da19c1933#r68860743>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABBTOaknlZwkELgc7oL0pM5DVDWVCWkks5qQarCgaJpZM4I_WMp>
> .
>